helpers/hasPassedLesson.ts

```diff
@@ -1,17 +1,15 @@
-import { UserLesson } from './dbload'
-import _ from 'lodash'
-import { LessonStatus } from '../@types/lesson'
+import { prisma } from '../prisma'
 
 export const hasPassedLesson = async (
   reviewerId: number,
   lessonId: string
 ): Promise<Boolean> => {
   // query userlesson that belongs to lesson and user
-  const userLesson = (await UserLesson.findOne({
+  const userLesson = await prisma.userLesson.findFirst({
     where: {
-      lessonId: lessonId,
+      lessonId: Number(lessonId),
```

Comment:
Are there plans to update grapql mutation as well? It's weird that Prisma already uses Int as type while resolver asks for string as argument.

Reply:
Yes, eventually. When I get to the submissions resolvers this will be fixed, since this is a helper function used there.